include/eeprom/core/hardware_iface.hpp

```diff
@@ -3,12 +3,20 @@
 namespace eeprom {
 namespace hardware_iface {
 
+static const size_t EEPROM_ADDR_8_BIT = sizeof(uint8_t);
```

Comment:
can we make these special elements of an `enum class` or something so they can be typed as like `EnumAddressType`